package.json

```diff
@@ -197,7 +197,7 @@
     "@balena/es-version": "^1.0.0",
     "@oclif/command": "^1.8.0",
     "@resin.io/valid-email": "^0.1.0",
-    "@sentry/node": "^5.21.1",
+    "@sentry/node": "^5.25.0",
```

Comment:
I have not actually seen any evidence that this version upgrade fixes issue #2060, but one can always hope... Even if it doesn't, the introduction of the `BALENARC_NO_SENTRY` env var should aid in debugging.